src/server/DRL_parameters.py

```diff
@@ -0,0 +1,25 @@
+class DRL_parameters:
+    def __init__(self,
+        nb_epochs: int,
+        learning_rate: float,
+        batch_size: int,
+        momentum: float) :
+        self.__nbEpochs = nb_epochs
+        self.__learningRate = learning_rate
+        self.__batchSize = batch_size
+        self.__momentum = momentum
+
+    def get_parameters(self):
+        return {
+            "nbEpochs" : self.__nbEpochs,
+            "learningRate" : self.__learningRate,
+            "batchSize" : self.__batchSize,
+            "momentum" : self.__momentum
+        }
+    
+    def set_parameters(self, params):
```

Comment:
It would be better to split this method in multiple smaller ones.

Reply:
why ? 
I don't need it in my part-code personally. But it can be an improvement point, no problem.

Reply:
> why ?

For several reason :
- If someone ever need to change a single parameter at a time he will need to change all of them
- If you add another parameter, all calls to this methods will have to be updated
- For a better separation of concerns : a single method handling everything is not always the best approach. The limits between what should and what should be separated is an open discussion :)

Reply:
ok we will discuss it tomorrow.